homeassistant/components/pi_hole/const.py

```diff
@@ -17,3 +19,9 @@
 SERVICE_DISABLE_ATTR_DURATION = "duration"
 
 MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
+
+VERSION_6_RESPONSE_TO_5_ERROR = {
```

Comment:
I understand the reason for this being defined here like this, and that it being here is nice to have rather than essential, but it'd be nice if there was some link to [the source in FTL](https://github.com/pi-hole/FTL/blob/88737f6248cd3df3202eed72aeec89b9fb572631/src/webserver/lua_web.c#L83) which generated this error for reference, and to use a pattern match on the string, since `pi.hole` is substituted for the configured hostname.